etc/kayobe/kolla.yml

```diff
@@ -94,6 +94,51 @@ kolla_openstack_release: "6.0.0.0"
 
 # Dict mapping Jinja2 block names in kolla's Docker images to their contents.
 #kolla_build_blocks:
+kolla_build_blocks:
+  #fluentd_footer: |
+  #  # Install a custom fluentd output plugin that forwards logs to the monasca
+  #  # log API.
+  #  ADD https://github.com/stackhpc/fluentd-monasca/archive/master.tar.gz /fluentd-monasca.tar.gz
+  #  RUN mkdir /fluentd-monasca \
+  #      && cd /fluentd-monasca \
+  #      && tar -zxf /fluentd-monasca.tar.gz \
+  #      && rm /fluentd-monasca.tar.gz
+  #  RUN cd /fluentd-monasca/fluentd-monasca-master \
+  #      && gem build fluentd-monasca-output.gemspec \
+  #      && gem install fluentd-monasca-output-0.0.1.gem \
+  #      && fluent-gem install fluentd-monasca-output-0.0.1.gem \
+  #      && rm -rf /fluentd-monasca
+  grafana_footer: |
+    # TODO: Re-enable the official plugin when #450868 is merged upstream. For now we will
+    # manually install it from git
+    # RUN grafana-cli plugins install monasca-datasource
+    RUN mkdir -p /var/lib/grafana/plugins/monasca-datasource \
+        && curl -Lo tmp.tgz https://github.com/stackhpc/monasca-grafana-datasource/tarball/log-query-api \
+        && tar zxvf tmp.tgz -C /var/lib/grafana/plugins/monasca-datasource --strip-components=1 \
+        && rm tmp.tgz
+    # Install plugin for Gantt charts
+    RUN grafana-cli plugins install natel-discrete-panel
+    RUN grafana-cli plugins install vonage-status-panel
+  ironic_inspector_footer: |
+    # Install our custom inspector plugins.
+    RUN pip install stackhpc-inspector-plugins
```

Comment:
How splendid, I didn't realise we were registered on pypi, nice work...

Reply:
Professional outfit here y'know.